server/views/pages/assessEmail.njk

```diff
@@ -65,6 +63,25 @@
                                 text: 'Enter a valid date and time in the past'
                             } if errors.recallNotificationEmailSentDateTime else undefined
                         }) }}
+                        {{ govukFileUpload({
+                            id: "recallNotificationEmailFileName",
+                            name: "recallNotificationEmailFileName",
+                            label: {
+                                text: "Upload the email"
+                            },
+                            hint: {
+                                html: govukInsetText({
+                                    text: "Save the email to your computer then upload the file."
+                                })
+                            },
+                            attributes: {
+                                accept: '.msg'
```

Comment:
do we have a test to show that you cant upload a non '.msg' file?

Reply:
no, I'll add one